marimo/_server/file_manager.py

```diff
@@ -8,28 +8,35 @@
 from marimo._ast import codegen
 from marimo._ast.app import App, InternalApp, _AppConfig
 from marimo._ast.cell import CellConfig
+from marimo._config.config import MarimoConfig
 from marimo._runtime.layout.layout import (
     LayoutConfig,
     read_layout_config,
     save_layout_config,
 )
 from marimo._server.api.status import HTTPException, HTTPStatus
-from marimo._server.models.models import (
-    SaveRequest,
-)
+from marimo._server.models.models import SaveRequest
 from marimo._server.utils import canonicalize_filename
 
 LOGGER = _loggers.marimo_logger()
 
 
 class AppFileManager:
-    def __init__(self, filename: Optional[str]) -> None:
+    def __init__(
+        self,
+        filename: Optional[str],
+        user_config: MarimoConfig | None = None,
```

Comment:
Not sure if this should be a required argument. Pyodide `save_file()` (`bootsrap.py`) for example uses `AppFileManager` but doesn't have access to the user config (and doesn't need it to save the file).

Reply:
it probably doesn't need to take in the full `user_config` right? maybe just `initial_width`?